.travis/script/mingw.sh

```diff
@@ -1,7 +1,8 @@
 ./autogen.sh &&
-export CPPFLAGS="-I/$HOME/build/win32/libyaml/include/" &&
-export LDFLAGS="-L/$HOME/build/win32/libyaml/lib/" &&
-./configure --host i586-mingw32msvc $ENABLE_UCS4 --with-yaml --prefix=$(pwd)/out-mingw32-install &&
+export CPPFLAGS="-I/$HOME/build/win$ARCH/libyaml/include/" &&
+export LDFLAGS="-L/$HOME/build/win$ARCH/libyaml/lib/" &&
+./configure --host $(test $ARCH == 32 && echo i586-mingw32msvc || echo x86_64-w64-mingw32) \
+            $ENABLE_UCS4 --with-yaml --prefix=$(pwd)/out-mingw32-install &&
```

Comment:
It seems a bit odd to leave the install prefix an `out-mingw32-install`, but I see that it is hard coded in other places and is basically just a temp dir. Maybe use a more generic name for it. Not so important though.

Reply:
Yeah it's indeed just a temp dir. I'll change the name and maybe incorporate `$ARCH` in it.

Reply:
Cool